test/cpp/jit/CMakeLists.txt

```diff
@@ -97,8 +97,6 @@ elseif(USE_ROCM)
     ${PYTORCH_HIP_HCC_LIBRARIES}
     ${TORCH_CUDA_LIBRARIES})
 
-  target_link_libraries(test_jit PRIVATE caffe2_gpu)
```

Comment:
qq: why getting rid of this one? is this removal somehow related to enabling torchbind test?

Reply:
It's not required for the torchbind test, but so it is the twin of the line we need to remove for the torchbind test - the reason being that caffe2_gpu does not exist anymore.

Reply:
Thank you for looking at this!